imports/plugins/included/wallet/server/methods/wallet.js

```diff
@@ -0,0 +1,205 @@
+        transactionId: transaction.id,
+        response: {
+          amount: total,
+          transactionId: transaction.id,
+          currency: paymentData.currency
+        }
+      };
+    } catch (error) {
+      Logger.warn(error);
+      result = {
+        saved: false,
+        error
+      };
+    }
+    return result;
+  },
+
+  /**
+    * Capture a Charge
+    * @param {Object} paymentData Object containing data about the transaction to capture
+    * @return {Object} results normalized
+  */
+  "wallet/payment/capture": function (paymentData) {
+    check(paymentData, Reaction.Schemas.PaymentMethod);
+    const authorizationId = paymentData.transactionId;
+    const amount = paymentData.amount;
+    const response = WalletApi.methods.capture.call({
+      authorizationId,
+      amount
+    });
+    const result = {
+      saved: true,
+      response
+    };
+    return result;
+  },
+
+  /**
+    * Create a refund
+    * @param  {Object} paymentMethod object
+    * @param  {Number} amount The amount to be refunded
+    * @return {Object} result
+  */
+  "wallet/refund/create": function (paymentMethod, amount) {
+    check(paymentMethod, Reaction.Schemas.PaymentMethod);
+    check(amount, Number);
+    const {
+      transactionId
+    } = paymentMethod;
+    const response = WalletApi.methods.refund.call({
+      transactionId,
+      amount
+    });
+    const results = {
+      saved: true,
+      response
+    };
+    return results;
+  },
+
+  /**
+    * List refunds
+    * @param  {Object} paymentMethod Object containing the pertinant data
+    * @return {Object} result
+  */
+  "wallet/refund/list": (paymentMethod) => {
+    check(paymentMethod, Reaction.Schemas.PaymentMethod);
+    const {
+      transactionId
+    } = paymentMethod;
+    const response = WalletApi.methods.refunds.call({
+      transactionId
+    });
+    const result = [];
+    for (const refund of response.refunds) {
+      result.push(refund);
+    }
+
+    // The results retured from the GenericAPI just so happen to look like exactly what the dashboard
+    // wants. The return package should ba an array of objects that look like this
+    // {
+    //   type: "refund",
+    //   amount: Number,
+    //   created: Number: Epoch Time,
+    //   currency: String,
+    //   raw: Object
+    // }
+    const emptyResult = [];
+    return emptyResult;
+  },
+  "wallet/getWalletDetails": (userId) => {
+    check(userId, String);
+    return Wallets.find({ userId }).fetch();
+  },
+  "wallet/update-balance": async function (transaction) {
+    check(transaction, Object);
+    const {
+      amount,
+      to,
+      from,
+      transactionType
+    } = transaction;
+
+    const updateBalance = (currentBalance, updateAmount, ownerEmail) => {
+      try {
+        Wallets.update({ ownerEmail }, {
+          $set: {
+            balance: currentBalance + updateAmount
+          }
+        });
+        return 1;
```

Comment:
Please change this to `return 'Success'`. Also change line 161 to `return 'Error'`. Let this change reflect in lines 59 and 61 of `completeOrder.js`

Reply:
I have implemented the requested changes.